rosdep/python.yaml

```diff
@@ -3096,9 +3096,10 @@ python-pylint:
   ubuntu: [pylint]
 python-pylint3:
   debian: [pylint3]
-  fedora: [python3-pylint]
   gentoo: [dev-python/pylint]
-  ubuntu: [pylint3]
+  ubuntu:
+    '*': null
```

Comment:
It looks like `pylint3` was renamed to `pylint`. I think we should make this `pylint` instead of `null` since that is its name in Focal and beyond..

* `pylint3` was transitioned with a dummy package with a dependency on new name in Debian: https://packages.debian.org/bullseye/pylint3
* pylint going forward in Debian: https://packages.debian.org/sid/pylint
* Ubuntu link: https://packages.ubuntu.com/focal/pylint

Reply:
Beat me to it - I was just combing thru ubuntu package listings :) I can second that this is also the conclusion I came to